src/runtime/cmd/kata-runtime/kata-check.go

```diff
@@ -441,6 +545,12 @@ EXAMPLES:
 			fmt.Println(successMessageCreate)
 		}
 
+		err = checkVersions()
+
```

Comment:
No blank line between `err = ` and `if err != nil`, also here you can only `return err` regardless of what the value of `err`.